src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -1647,6 +1647,9 @@ void Unit::CalcAbsorbResist(Unit* victim, SpellSchoolMask schoolMask, DamageEffe
 
     RoundToInterval(auraAbsorbMod, 0.0f, 100.0f);
 
+    auto absorb_ignoring_damage = CalculatePct(dmgInfo.GetDamage(), auraAbsorbMod);
```

Comment:
auto is supposed to replace long-to-type types like std::vector<Creature*>::const_iterator, in this case instead it just replaces a simple uint32. it also makes more difficult to see if the type is a signed/unsigned integer or a float/double.